Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,127 @@
 import React from 'react';
 import './App.css';
+import Display from './components/DisplayComponents/CalculatorDisplay';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton'
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
-
-export default App;
+const numbers = [
+  {
+      text: '7',
+      buttonStyle: 'number'
+  },
+  {
+      text: '8',
+      buttonStyle: 'number'
+  },
+  {
+      text: '9',
+      buttonStyle: 'number'
+  },
+  {
+      text: '4',
+      buttonStyle: 'number'
+  },
+  {
+      text: '5',
+      buttonStyle: 'number'
+  },
+  {
+      text: '6',
+      buttonStyle: 'number'
+  },
+  {
+      text: '1',
+      buttonStyle: 'number'
+  },
+  {
+      text: '2',
+      buttonStyle: 'number'
+  },
+  {
+      text: '3',
+      buttonStyle: 'number'
+  },
+];
+
+const actions = [
+  {
+    text: 'Clear',
+    buttonStyle: 'action'
+  },
+  {
+    text: '0',
+    buttonStyle: 'action',
+  },
+]
+
+const functions = [
+  {
+    text: '÷',
+    buttonStyle: 'function',
+    value: '/'
+  },
+  {
+    text: '×',
+    buttonStyle: 'function',
+    value: '*'
+  },
+  {
+    text: '−',
+    buttonStyle: 'function',
+    value: '-'
+  },
+  {
+    text: '+',
+    buttonStyle: 'function',
+    value: '+'
+  },
+  {
+    text: '=',
+    buttonStyle: 'function',
+    value: '='
+  }
+]
+
+class App extends React.Component {
+
+  state = {
+    output: '0'
+  };
+
+  inputDigit(digit) {
+    const {output} = this.state;
+
+    this.setState({
+      output: output === '0' ? String(digit) : output + digit
+    })
+  } 
+
+  render() {
+
+    return (
+      <div>
+        <Display output={this.state.output}/>
+        <div className="flex">
+  
+          <div className="buttonwrap">
+            
+            <ActionButton action={actions[0]}/>
+            
+            <div className='numbers'>
+              {numbers.map(number => {return <NumberButton onClick={()=>{this.inputDigit(number)}} number={number}/>;})}
```

Comment:
After learning how to use onClick with state lecture, you just need to put it in the components itself.